brick/__brick__/{{project_name.snakeCase()}}/.github/workflows/main.yaml

```diff
@@ -10,4 +10,4 @@ jobs:
     uses: VeryGoodOpenSource/very_good_workflows/.github/workflows/flutter_package.yml@v1
     with:
       flutter_channel: stable
-      flutter_version: 3.7.0
+      flutter_version: 3.7.1
```

Comment:
```suggestion
      flutter_version: 3.7.3
```

Reply:
I was unsure if it would be better to get the current version (3.7.3) or just meet the cli requirements.
I've just updated the PR.